packages/utils/tool-utils/src/snapshotNormalizer.ts

```diff
@@ -91,6 +94,18 @@ function getNormalizedBlobContent(blobContent: string, blobName: string): string
         content = JSON.stringify(gcState);
     }
 
+    /**
+      * The metadata blob has "summaryNumber" that tells which summary this is for a container. This can be different in
+      * summaries of two clients even if they are generated at the same sequence number. For instance, at seq# 1000, if
+      * one client has summarized 10 times and other has summarizer 15 times, summaryNumber will be different for them.
+      * So, update "summaryNumber" to 0 for purposes of comparing snapshots.
+      */
+    if (blobName === metadataBlobName) {
```

Comment:
The snapshot tests runs the replay tool in modes where two containers load from the same snapshot but generate summaries at different rates. Meaning that two containers that generate snapshot at a given seq# may have generated different number of snapshots before this. Hence, the need to normalize the summaryNumber.